distributions/Dist_jal849.py

```diff
@@ -0,0 +1,22 @@
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_jal849(BaseDistribution):
+	def __init__(self):
+		self.f_max = 8
+		self.x_min = -1
+		self.x_max = 1
+
+
+	def pdf(self, x):
+		"""This is your PDF"""
+		return -(x^2)+8
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 23/3
```

Comment:
23/3 will result in integer division and give you 7. 
Instead 23./3=7.666666666666667 
(notice the . to make 23. a floating point number).